### DISCUSSION TITLE
PatchTool 7.0 6/28

### DISCUSSION THREAD

**Discussion by GpointChen** [GpointChen/FFXIVChnTextPatch-GP#442] [Announcements]
- 7.0相容性更新。
- 漢化器（.exe）更新。
- 7.0部分機翻漢化。文字尾部加有「★」代表這部分是機翻。
- 文本內容量大，目前僅對一小部分內容（主要是與遊戲指令碼相關的部分）做了人工檢查。如果仍出現遊戲跳錯或崩潰，請盡可能紀錄案發現場和使用設定。

請從release下方連結或內文連結下載檔案。以下兩種方法擇一使用即可。

## 方法一：直接覆蓋遊戲檔案

解壓縮後直接覆蓋`SquareEnix\FINAL FANTASY XIV - A Realm Reborn\game\sqpack\ffxiv`內的同名檔案即可。內含三個000000開頭的字體相關檔案與三個0a0000開頭的文本相關檔案。

設定：Windows作業系統，非高清字體，以繁體中文覆蓋日文和英文，官方渠道版本。

設定相同的人可以考慮直接下載這包，（可能）比較不會遇到問題。其他作業系統未測試。

- ffxiv-20240628.rar：請從下方連結下載。

## 方法二：使用漢化器

使用方法請參照[readme](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/2.-%E4%BD%BF%E7%94%A8)。

檔案：

- FFXIVChnTextPatch-20240628.rar：漢化器本體。需要自行調整設定的人請下載這包手動漢化。請從下方連結下載。

視需求選用：

- [font-custom-4k-smaller-20230131.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v3.0.1/font-custom-4k-smaller-20230131.rar)：高畫質字體包，請取代原本的resource/font資料夾。[使用此字體包需更改設定。](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94)透過重複使用字體字型縮小檔案體積，使用Noto Sans。
- [font-custom-20230111.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.72.2/font-custom-20230111.rar)：舊高畫質字體包，請取代原本的resource/font資料夾。[使用此字體包需更改設定。](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94)所有大小字體皆分開繪製，使用Noto Sans和Noto Serif。
- rawexd-zhcn-20240628.rar：簡體中文CSV，取代原本resource/rawexd資料夾。
- [FFXIVChnTextPatch-20240319.jar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v3.2.1.5/FFXIVChnTextPatch-20240319.jar)：如果你有遇到JNI error，請參考 https://github.com/GpointChen/FFXIVChnTextPatch-GP/issues/90 放到與exe同位置使用。
- backup-20240628.rar：乾淨的0X0000檔案。如果漢化出現問題，可以將這些檔案放到backup後按還原（或是手動取代遊戲檔案），然後再次嘗試漢化。只適用於這個release發布時的遊戲版本。

※若不替換字體會導致遊戲內無法正常顯示部分中文字。另外，目前大廳部分中文字必須使用高畫質字體包才能正常顯示。更多資訊可參閱[wiki頁面](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94)。

有問題歡迎至Issues或Discussions回報。

<hr /><em>This discussion was created from the release <a href='https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/tag/v3.3.0.0'>PatchTool 7.0 6/28 </a>.</em>

**Comment by ETplanet** [comment]
大大威武神速！感恩！
請問漢化器的檔案是不是沒放到?
沒看到FFXIVChnTextPatch-20240628.rar這個檔案
還是目前暫時只能以取代的方式處理?

**Reply by GpointChen** [reply]
漏放了！待會補

**Reply by ETplanet** [reply]
謝大大，您辛苦了！！！！

**Comment by AnnChen516** [comment]
謝謝^^辛苦了

**Comment by Manbbb** [comment]
感謝你，這速度真的辛苦你了

**Comment by corey924** [comment]
神速，非常感謝！

**Comment by bboy0607** [comment]
感謝大神!讓我們有遊戲玩~~~

**Comment by smallcarking** [comment]
![image](https://github.com/GpointChen/FFXIVChnTextPatch-GP/assets/18345707/2920ff4b-3313-4e37-91f9-0ada6ff9d1b1)
感謝大神！
但我遇到問題,差不多所有的技能介紹都是原文,未能漢化
特此告知

**Reply by GpointChen** [reply]
正常現象，可參考 #444

**Comment by CharReo** [comment]
大神日文漢化完之後 , 繪靈法師職務任務NPC會消失

**Comment by CharReo** [comment]
日文漢化後進無人島斷線

**Comment by Cleffl** [comment]
Knowing the hanuhanu這個需要說話的任務似乎有點問題 用英文客戶端漢化後用英文回答沒辦法完成 任務提示也顯示成日文
漢化包拔了就能用英文回答解了

**Comment by HollowNite** [comment]
請問貨幣顯示不出來變黑黑一塊是正常的嗎
![image](https://github.com/GpointChen/FFXIVChnTextPatch-GP/assets/98972304/8c433163-3932-430a-baf6-e58640e2caf6)

**Reply by chun1324** [reply]
這是官方的問題, 對他按下右鍵切換至其他課幣

公告文
https://na.finalfantasyxiv.com/lodestone/news/detail/d6a632f3c438f8ef92954f30af4d3cb1b79af011

**Reply by HollowNite** [reply]
我試過了，可是沒有反應

**Comment by YoLoN23** [comment]
我漢化後用不了獅子種族的傭人,還原日文就能用,是漢化有小小BUG嗎?

**Comment by loier3600** [comment]
目前繪靈法師的83等特職任務無法觸發，發任務的莫古力直接消失，我試著用英文去觸發任務後再漢化，但結果變成在接下這個任務的情況下所有任務欄都失效漂白，只能放棄該任務。  我不知道要是我完成這個任務後再漢化會發生什麼事情，就先擱置了(反正現在沒有擋到職能任務就好。)

**Reply by CURT0509** [reply]
我剛才也發生一樣的問題 而且任務列表會直接消失,還沒辦法還原日文版-.-

**Reply by chun1324** [reply]
> 我剛才也發生一樣的問題 而且任務列表會直接消失,還沒辦法還原日文版-.-

只要覆蓋備份檔即可還原, 如果失敗就再確認安裝位置以及資料夾名稱
